devimint/src/external.rs

```diff
@@ -845,6 +864,7 @@ pub async fn open_channel(
         .context("bech32 should be present")?;
 
     bitcoind.send_to(cln_addr, 100_000_000).await?;
+    bitcoind.wait_mempool_size(1).await?;
```

Comment:
Could be flaky, if there's anything that will trigger the block in the meantime.

Reply:
Also, I doubt it's bitcoind dropping things.